.github/workflows/build-and-publish.yaml

```diff
@@ -0,0 +1,136 @@
+---
+name: Build and Publish Images
+
+on:
+  push:
+    branches:
+      - redhat-3.5
+
+jobs:
+  quay-image:
+    name: Calculate Quay Image Digest
+    runs-on: 'ubuntu-latest'
+    outputs:
+      digest: ${{ steps.set-output.outputs.digest }}
+    env:
+      IMAGE_REGISTRY: quay.io/projectquay
+      Y_STREAM: 3.5
```

Comment:
should we just make this TAG? that way we can include things like latest, nightly, etc.

Reply:
good idea!

Reply:
As for the value itself, do we want `3.5`, `v3.5`, or even `redhat-3.5`?
To me the only important thing is that we're consistent throughout repositories. If any project uses different versions for upstream vs downstream then we might want to go with `redhat-3.5`, but I don't know if this is the case (maybe clair does this?)

Reply:
Using the branch name (`redhat-3.5`) also has the benefit of it being easy to automate for different branches, I think github actions provides means of retrieving that via environment variable.